### ISSUE TITLE
Doc : Missing ddoc parameters 

### ISSUE THREAD

**Issue by cl-parsons** [apache/couchdb#1759]
On [this page of the documentation](http://docs.couchdb.org/en/stable/ddocs/ddocs.html?highlight=updatefun) there is no information about the keys we should use to specify these functions.

In my case I was looking how to custom the data submitted to the view by adding a timestamp to the object. I found how to do this in [the "best practice" page](http://docs.couchdb.org/en/stable/best-practices/forms.html#the-update-function), but I think that this should be also specified in the API reference.

**Comment by wohali** [comment]
@cl-parsons I agree, this is a shortcoming. If you'd like to help us, we'd :heart: a pull request from a passionate user.
